### PR TITLE
Use isort to automatically sort Python imports

### PR DESCRIPTION
**Pull request by jdufresne**
isort is a Python utility to sort imports alphabetically by section. It
improves the contributor experience by catching style concerns early in
CI rather than during a review. When CI fails, the contributor can use
the tool to apply changes automatically.

isort is very popular and used by big Python projects such as pip.

https://pycqa.github.io/isort/

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -19,3 +19,6 @@ classifiers = [
     "Programming Language :: Python :: 2.7",
     "Programming Language :: Python :: 3",
 ]
+
+[tool.isort]
+profile = "black"
```
Comment (pradyunsg) [inline pyproject.toml:24]: You also need to set the multiline output line mode.

https://pycqa.github.io/isort/docs/configuration/black_compatibility.html#using-a-config-file-such-as-isortcfg

**Review by pradyunsg — DISMISSED** [review]

File: pyproject.toml (hunk 1 above)
Comment (jdufresne) [reply pyproject.toml:24]: According to the isort docs here: https://pycqa.github.io/isort/docs/configuration/profiles.html

That is `multi_line_output = 3` is included in the black profile. Did you mean something else?

Comment (jdufresne) [reply pyproject.toml:24]: Confirmed with the isort author that adding `multi_line_output` is unnecessary with the profile:

https://github.com/PyCQA/isort/pull/1858

Comment (pradyunsg) [reply pyproject.toml:24]: Neato! Thanks for doing this. I was just looking to make the same PR. :)

**Review by pradyunsg — APPROVED** [review]

**Pull request merged**